src/catalog/tests/debug.rs

```diff
@@ -88,8 +89,38 @@ async fn test_debug(
     assert_eq!(Epoch::new(2).unwrap(), epoch);
 
     // Check opened trace.
-    let trace = openable_state2.trace().await.unwrap();
+    let mut trace = openable_state2.trace().await.unwrap();
+    let (idx, user_version) = {
+        // The user_version changes frequently so we test the value in rust and remove it from the
+        // trace to avoid changing it in the snapshot file.
```

Comment:
I love the idea! The insta snapshot operates on the Debug impl of the trace. What do you think about making a test helper that returns the user version for comparison, and a newtype struct that wraps the trace and has a debug impl that ignores the user version? This way you don't have to worry about removing and re-inserting it

Reply:
Great idea! I just updated the PR to do this.